docs/dev_guide/contents/pr_review_procedure.rst

```diff
@@ -25,11 +25,19 @@ Before the "merge" button is clicked the following criteria must be met:
 
 * All the continuous integration must pass unless there is a known issue.
 
-* At least two members (not the author of the PR) of the "sunpy-developers" group must have approved the PR, one should, ideally, be a relevant subpackage maintainer.
+* At least two members (excluding the PR author) of the "sunpy-developers" group must have approved the PR.
+  One should, ideally, be a relevant subpackage maintainer.
```

Comment:
I wonder if this line is still relevant outside of coordinates.

Reply:
Yeah, we got rid of subpackage maintainers right? I could just remove this line?

Reply:
I think it should go.